library/src/main/java/com/novoda/downloadmanager/notifications/NotificationsCreatedListener.java

```diff
@@ -0,0 +1,31 @@
+package com.novoda.downloadmanager.notifications;
+
+import android.app.Notification;
+import android.app.Service;
+import android.support.v4.util.SimpleArrayMap;
+
+import static com.novoda.downloadmanager.notifications.SynchronisedDownloadNotifier.TYPE_ACTIVE;
+
+public class NotificationsCreatedListener {
+    private Service service;
+
+    public NotificationsCreatedListener(Service service) {
```

Comment:
if it has to stay public, can we javadoc it then? (even if its just to say _not expected to be used by clients_)